src/components/FilterTop.js

```diff
@@ -230,7 +230,7 @@ function Filter(props) {
                   format={getDateFormatLocale(true)}
                   value={dateStart}
                   onChange={handleDateStartChange}
-                  maxDate={dateEnd}
+                  maxDate={dateEnd || Date('2100-01-01')}
```

Comment:
Same as with Filter.js we can set defaults on lines 72-73 so they are set from the start.